src/main/java/com/cozyhome/onlineshop/userservice/controller/AuthController.java

```diff
@@ -51,7 +51,7 @@ public class AuthController {
 	private final String emailErrorMessage = "Error: Email is already in use!";
 	private final String registrationSuccessMessage = "User registered successfully!";
 
-	@Operation(summary = "Existing user login", description = "Existing user login by email and password")
+	@Operation(summary = "User login", description = "Allows an existing user to log in using his email and password.")
```

Comment:
I'd leave that line as it was: summary = "Existing user login", because it concretizes and in "description" we use this word to.